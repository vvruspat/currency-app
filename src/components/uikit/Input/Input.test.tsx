import { fireEvent, render, screen } from "@testing-library/react";
import { Input } from ".";

test("renders input", () => {
  render(<Input data-testid="test-input" />, {});
  const InputElement = screen.getByTestId("test-input");
  expect(InputElement).toBeInTheDocument();
  expect(InputElement).toMatchSnapshot();

  fireEvent.change(InputElement, { target: { value: "Input Test Value" } });

  const InputElementUpdated = screen.getByDisplayValue("Input Test Value");
  expect(InputElementUpdated).toBeInTheDocument();
  expect(InputElementUpdated).toMatchSnapshot();
});
