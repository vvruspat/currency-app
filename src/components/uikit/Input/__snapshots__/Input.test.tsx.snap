// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders input 1`] = `
<input
  class="input input-align-left"
  data-testid="test-input"
/>
`;

exports[`renders input 2`] = `
<input
  class="input input-align-left"
  data-testid="test-input"
/>
`;
